### PR TITLE
Let server documentation be first and add plugins

### PR DESCRIPTION
**Pull request by petterarvidsson**
@nsg I wanted to add a list of plugins to the web page. I think a table is a nice format (I am open for other formats as well), but I am not sure where I should add styling for the table. It looks quite terrible with the current styling.

### CODE REVIEW

**Comment by nsg** [comment]
`_layouts/style.css` is a good place to add custom CSS styles, I think. Let me know if you need any pointers.

File: _posts/2016-10-18-release-7.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- Protocol update due to block rotation
+- Improved block query code
+- Lots of updates to plugin API, latest docs available [here](http://doc.konstructs.org/server-api/index.html)
+
+### Plugins
+
+- [Forest plugin](https://github.com/konstructs/server-plugin-forest) now allows rotation of wood blocks
+- [Ore plugin](https://github.com/konstructs/server-plugin-ore) new plugin!
+- [Ore plugin](https://github.com/konstructs/server-plugin-caves) new plugin!
```
Comment (nsg) [inline _posts/2016-10-18-release-7.md]: You forgot to update the link description.

**Review by nsg — COMMENTED** [review]
Why?

**Comment by nsg** [comment]
Oh, a review is apparently always on the full PR, not the single commit I was viewing ... I was referring to https://github.com/konstructs/konstructs.github.io/pull/11/commits/ee6ab9ec70a52e6df0bb1b5ec774046130d200c3

**Comment by petterarvidsson** [comment]
@nsg please see https://github.com/konstructs/konstructs.github.io/pull/9, but basically because it is impossible to set a class using MD

**Comment by petterarvidsson** [comment]
@nsg Maybe like this?

**Comment by nsg** [comment]
:+1: for `{: .table}` :)

**Pull request merged**